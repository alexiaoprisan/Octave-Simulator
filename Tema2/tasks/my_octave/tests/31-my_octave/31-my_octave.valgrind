==41525== Memcheck, a memory error detector
==41525== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==41525== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==41525== Command: /home/dominic/Downloads/check_my_octave/my_octave
==41525== Parent PID: 41524
==41525== 
==41525== 
==41525== HEAP SUMMARY:
==41525==     in use at exit: 0 bytes in 0 blocks
==41525==   total heap usage: 299 allocs, 299 frees, 12,740 bytes allocated
==41525== 
==41525== All heap blocks were freed -- no leaks are possible
==41525== 
==41525== For lists of detected and suppressed errors, rerun with: -s
==41525== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
