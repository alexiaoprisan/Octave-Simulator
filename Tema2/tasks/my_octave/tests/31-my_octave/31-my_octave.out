Cannot perform matrix multiplication
Unrecognized command
Unrecognized command
Unrecognized command
Unrecognized command
Unrecognized command
Unrecognized command
3 3
Power should be positive
7461 2006 6558 
7782 2050 6325 
8103 2094 6092 
3 3
